docs/book/basics/creating_events.md

```diff
@@ -0,0 +1,50 @@
+{% panel style="info", title="Under Development" %}
+This book is being actively developed.
+{% endpanel %}
+
+# Creating Events
+
+It is often useful to publish *Event* objects from the controller Reconcile function.  Events
+allow users to see what is going on with a particular object, and allow automated processes
+to see and respond to them.
+
+{% panel style="success", title="Getting Events" %}
+Recent Events for an object may be viewed by running `kubectl describe`
+{% endpanel %}
+
+{% method %}
+
+Events are published from a controller using an event recorder, which is automatically configured
+by `kubebuilder create resource`.  The event recorded is intended to be used from the `Reconcile`
+function.
+
+```go
+Event(object runtime.Object, eventtype, reason, message string)
```

Comment:
Can you replace `eventtype` by `type` to be consistent with subsequent lines?

Reply:
Done

Reply:
type is a reserved word, so changing the doc to eventtype